src/points.rkt

```diff
@@ -207,6 +227,18 @@
            (length e))
         (apply max (map ulps->bits reals)))))
 
+(define (errors prog pcontext)
+  (let ([fn (eval-prog prog 'fl)]
+	[max-ulps (expt 2 (*bit-width*))])
+    (for/list ([(point exact) (in-pcontext pcontext)])
+      (let ([out (fn point)])
+	(add1
+	 (if (or (real? out)
```

Comment:
Off the top of your head, what can this be if not `real?`? Is it from back in the day when this sometimes contained complex numbers accidentally? I don't think `real?` filters out NaNs, for example.